src/main/java/io/seqera/tower/cli/responses/computeenvs/ComputeEnvList.java

```diff
@@ -38,9 +48,14 @@ public void toString(PrintWriter out) {
             return;
         }
 
-        TableList table = new TableList(out, 5, "ID", "Status", "Platform", "Name", "Last activity").sortBy(0);
+        TableList table = new TableList(out, 5, "  ID", "Status", "Platform", "Name", "Last activity").sortBy(0);
```

Comment:
Why the extra space in the ID header?

Reply:
To align the start of the header with the start of the cell. Here the cells have an extra `*` or ` ` at the beginning to mark the primary compute environment.

Reply:
Good enough, thanks